src/styles/main.scss

```diff
@@ -1,3 +1,386 @@
-body {
-  margin: 0;
+.container {
+  background-color: $main-color;
+
+  position: relative;
+
+  img {
+    width: 100%;
+
+    display: block;
+  }
+
+  .main {
+    padding: 60px 20px 20px;
+
+    img {
+      transition: transform 0.3s ease;
+
+      &:hover {
+        transform: scale(1.05);
+      }
+    }
+
+    h1,
+    h2,
+    legend {
+      font-size: 32px;
+      font-weight: 500;
+      line-height: 100%;
+    }
+
+    .title {
+      text-align: center;
+    }
+
+    .description {
+      font-size: 23px;
+      font-weight: 500;
+      line-height: 140%;
+
+      margin: 32px 0 92px;
+    }
+
+    .bike-card {
+      margin-bottom: 40px;
+
+      h3 {
+        font-size: 24px;
+        font-weight: 700;
+        line-height: 140%;
+      }
+
+      .bike-description {
+        font-size: 16px;
+        font-weight: 400;
+        line-height: 140%;
+
+        margin-top: 8px;
+        margin-bottom: 16px;
+      }
+    }
+
+    .details-card {
```

Comment:
Follow BEM strictly:

Rename classes like .description to .main__description for better scoping.
Avoid over-nesting inside blocks like .bike-card or .details-card. Use child elements explicitly, e.g., .details-card__bike-details.

```
.details-card {
  &__bike-details {
    display: flex;
    justify-content: space-between;
  }
}
```